src/nfts/ERC1155MaxSupplyMintable.sol

```diff
@@ -49,6 +50,20 @@ contract ERC1155MaxSupplyMintable is ERC1155Supply, ERC1155Burnable, CoreRef {
     /// @param tokenId the id of the token to update
     /// @param maxSupply the new max supply of the token
     function setSupplyCap(uint256 tokenId, uint256 maxSupply) external onlyRole(Roles.ADMIN) {
+        _setSupplyCap(tokenId, maxSupply);
+    }
+
+    /// @notice set the supply cap for a given token, cannot be less than current supply. Can only be called once by the DEPLOYER role
+    /// @param tokenId the id of the token to update
+    /// @param maxSupply the new max supply of the token
+    function setSupplyCapAtDeployment(uint256 tokenId, uint256 maxSupply) external onlyRole(Roles.DEPLOYER) sealAfter {
```

Comment:
Will need to update the unit and integration tests for `ERC1155MaxSupplyMintable.sol` to include `setSupplyCapAtDeployment`, but these can be done in another PR.